pnpm-lock.yaml

```diff
@@ -11160,7 +11187,7 @@ packages:
   /wide-align/1.1.5:
     resolution: {integrity: sha512-eDMORYaPNZ4sQIuuYPDHdQvf4gyCF9rEEV/yPxGfwPkRodwEgiMUUXTx/dex+Me0wxx53S+NgUHaP7y3MGlDmg==}
     dependencies:
-      string-width: 4.2.3
+      string-width: 1.0.2
```

Comment:
Whatsup with this?

Reply:
No idea tbh, I reset my dev environment and still get the same outcome.